UnitTests/PayPalTests/PayPalClient_Tests.swift

```diff
@@ -32,6 +32,9 @@ class PayPalClient_Tests: XCTestCase {
         func paypalDidCancel(_ payPalClient: PayPalClient) {
             paypalDidCancel = true
         }
+
+        func paypalDidStart(_ payPalClient: PayPalClient) {
+        }
```

Comment:
no-op or missing implementation?

Reply:
missing implementation, once we move forward with the tests

Reply:
Same comment re: adding a TODO